components/board.drugconnectivity/R/drugconnectivity_ui.R

```diff
@@ -4,87 +4,98 @@
 ##
 
 DrugConnectivityInputs <- function(id) {
-    ns <- shiny::NS(id)  ## namespace
-    bigdash::tabSettings(
-        withTooltip( shiny::actionLink(ns("dsea_info"), "Youtube", icon = shiny::icon("youtube") ),
-                "Show more information about this module."),
-        shiny::hr(), shiny::br(),             
-        withTooltip( shiny::selectInput(ns("dsea_contrast"),"Contrast:", choices=NULL),
-                "Select the contrast corresponding to the comparison of interest.",
-                placement="top"),
-        withTooltip( shiny::selectInput(ns('dsea_method'),"Analysis type:", choices = ""),
-                "Select type of drug enrichment analysis: activity or sensitivity (if available).",
-                placement="top")
+  ns <- shiny::NS(id) ## namespace
+  bigdash::tabSettings(
+    withTooltip(
```

Comment:
You can remove this button now that we have the big `i` on top of the page that triggers the same modal